src/main/kotlin/org/openmbee/flexo/sysmlv2/apis/CommitApi.kt

```diff
@@ -406,7 +508,7 @@ fun Route.CommitApi() {
 
         // parse the response model, convert it to JSON, and reply to client
         call.respond(flexoResponseUpdate.parseLdp {
-            commitFromModel(self!!, primary, UUID.fromString(projectId))
+            commitFromModel(focalIri!!, focalOutgoing, UUID.fromString(projectId))
```

Comment:
think this should be commitId? but commit id is created by flexo here..

Reply:
which thing? the focalIri will be set from Location response header which should be the flexo commit URI